### PR TITLE
D8 nid 1232

### PR DESCRIPTION
**Pull request by neilblair**

### CODE REVIEW

File: src/js/toggle-btn-script.js [hunk 1]
```diff
@@ -15,29 +15,33 @@
...
-          maxRel = parseInt($(this).attr('data-toggle-length'), 10 || 7),
-          relTotal = $relatedArtList.children('li').length,
-          available = relTotal - maxRel,
-          relToggleText = Drupal.t('Show @num more', {'@num': available});
+        let maxItems = parseInt($(this).attr('data-toggle-length'), 10);
+        if (isNaN(maxItems)) maxItems = 7;
 
-        if (relTotal > maxRel) {
-          $relatedArtList.attr({ id: 'toggle-menu', 'aria-live': 'polite' });
+        let $toggleList = $(this).find('ul'),
```
Comment (johangant) [inline src/js/toggle-btn-script.js:21]: What's the convention with JavaScript variables vs PHP variables ($ prefix)? Or does the $ here hint of a jQuery object?

Comment (johangant) [reply src/js/toggle-btn-script.js:21]: Just me thinking out loud here by the way!

**Review by johangant — APPROVED** [review]
Looks great. I had a small question around variable naming but certainly not a blocking issue.

File: src/js/toggle-btn-script.js (hunk 1 above)
Comment (neilblair) [reply src/js/toggle-btn-script.js:21]: Yep - there is a convention that variables containing jquery objects have $ prefix

Comment (neilblair) [reply src/js/toggle-btn-script.js:21]: https://stackoverflow.com/questions/6209462/when-why-to-prefix-variables-with-when-using-jquery

**Pull request merged**